src/yoti_common/index.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const crypto = require('crypto');
+const forge = require('node-forge');
+
+exports.getRSASignatureForMessage = (message, pem) => {
+  let sign = crypto.createSign('RSA-SHA256');
```

Comment:
We should have a standard for indentation. It seems we're using 2 spaces 4 spaces and tabs throughout the project. Maybe we should address this in a future PR.